doc/tutorials/content/sources/vfh_recognition/FindFLANN.cmake

```diff
@@ -1,21 +1,42 @@
-###############################################################################
```

Comment:
The existence of this file baffles me. I propose to remove it and add PCL's cmake modules folder to the `CMAKE_MODULE_PATH` path of this tutorial so that it makes use of the same file. This if accepted, to be done in a separate commit.

Reply:
Was wondering too ;)

Reply:
Hey. You marked this as resolved despite not being resolved yet, the duplicate file is still here. Just wanted to raise attention to this.

Reply:
Ah believed separate commit = separate PR. My fault ;).